sdk/eventhub/event-processor-host/src/hostContext.ts

```diff
@@ -154,25 +154,6 @@ export namespace HostContext {
     }
   }
 
-  function _eitherLeaseManagerOrleaseDurationAndRenewal(options: EventProcessorHostOptions): void {
-    validateType("options", options, true, "object");
-    const leaseManager = options.leaseManager;
-    const leaseDuration = options.leaseDuration;
-    const leaseRenewInterval = options.leaseRenewInterval;
-    if (leaseManager) {
```

Comment:
If user provides the lease manager, do we check anywhere that the lease manager has valid values for lease duration and renew interval?

Reply:
Yes, we validate for lease duration and renew interval right after we set those values.